config/connector_options.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+type ConnectorOptions func(*Options)
+
+type Options struct {
+	UseTracing bool
+
+	// unexported as these are specifically used by Hydra for writing tests
```

Comment:
👀

Reply:
This makes a ton of sense to me, all of these options are only used internally for testing and other stuff. Good job!

Reply:
Yes, that was completely my bad. I was not aware of the plugin use case. It makes sense to me 👍